nuxt/cypress/e2e/account.cy.ts

```diff
@@ -0,0 +1,17 @@
+describe('account test', () => {
```

Comment:
```suggestion
describe('account page', () => {
```
We are inside the test already and the test *describes* a page.